### DISCUSSION TITLE
Need help! Aboat video function!

### DISCUSSION THREAD

**Discussion by superhighsky** [Stability-AI/StableSwarmUI#281] [Q&A]
I update to the new version with the video option, put the SVD model into the stable-diffusion folder, but I can't show any model information when selecting the SVD model, please, how should the new video feature work?

**Comment by mcmonkey4eva** [comment]
You can refresh models and then _reload parameter values_ to update that:

![image](https://github.com/Stability-AI/StableSwarmUI/assets/4000772/91bba948-13f3-4aad-a7dd-97761230551e)


Or when in doubt when things aren't updating, just, restart swarm

**Reply by superhighsky** [reply]
thank you anyway
